survey form.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+	 <meta charset="utf-8">
+	 <title>Survey Form</title>
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+	 <link rel="stylesheet" href="surveyform.css">
+ </head>
+ <body>
+	 <h1 id="title">Microverse Survey Form</h1>
+	 <p id="description">The form aims to create the best student experience in Microverse</p>
+	  <div class="input-headings">
+		  <form id="survey-form">
+			  <label for="name" id="name-label">Name: &nbsp;</label><br>
+			  <input type="text" id="name" placeholder="Enter your name" required><br>
+			
+			  <label for="email" id="email-label">Email: &nbsp;</label><br>
+			  <input type="email" id="email" placeholder="Enter your email"><br>
+
+			  <label for="number" id="number-label">Age: &nbsp;</label><br>	
+			  <input type="number" id="number" placeholder ="Enter age btwn 18 and 40" min="18" max="40" class="form-inputs" required> 
+			  <br>		
+
```

Comment:
```suggestion
As for this part I suggest you use class instead of id:relaxed:, since in the part above there are 3 sections that refer to same form 
```